crates/router/src/core/errors/api_error_response.rs

```diff
@@ -148,6 +148,10 @@ pub enum ApiErrorResponse {
     MandateUpdateFailed,
     #[error(error_type = ErrorType::ObjectNotFound, code = "HE_02", message = "API Key does not exist in our records")]
     ApiKeyNotFound,
+    #[error(error_type = ErrorType::ValidationError, code = "HE_03", message = "Unable to find the mandate in our records.")]
```

Comment:
The error variant, and the message doesn't match can we try and match them?

Reply:
Making this "invalid mandate id passed from connector."